internal/applicationsnapshot/report_test.go

```diff
@@ -998,3 +999,184 @@ func createTestPolicy(t *testing.T, ctx context.Context) policy.Policy {
 	assert.NoError(t, err)
 	return p
 }
+
+func Test_DocumentationLink_OnlySuccesses(t *testing.T) {
+	// Test case: Only successes - should NOT show documentation link
+	r := Report{
+		ShowSuccesses: true,
+		Components: []Component{
+			{
+				Successes: []evaluator.Result{
+					{
+						Metadata: map[string]interface{}{
+							"code": "success.policy",
+						},
+						Message: "Policy passed successfully",
+					},
+				},
+				SuccessCount: 1,
+			},
+		},
+	}
+
+	output, err := generateTextReport(&r)
+	require.NoError(t, err)
+
+	outputStr := string(output)
+	hasDocLink := strings.Contains(outputStr, "https://conforma.dev/docs/policy/index.html")
+
+	assert.False(t, hasDocLink, "Documentation link should NOT appear when there are only successes")
+	t.Logf("✅ Correctly - no documentation link appears with only successes")
```

Comment:
It's subjective I guess, but I'd probably skip the output on success, i.e. all the t.Logf lines.